app/src/main/cpp/skyline/services/account/IProfile.h

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2020 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include "IAccountServiceForApplication.h"
+#include <services/base_service.h>
+
+namespace skyline::service::account {
+
+    /// UserData
+    typedef struct {
+        u32 unk_x0;                ///< Unknown.
+        u32 iconID;                ///< Icon ID. 0 = Mii, the rest are character icon IDs.
+        u8 iconBackgroundColorID;  ///< Profile icon background color ID
```

Comment:
You should probably make this an enum

Reply:
You mean an enum with different colors?

Reply:
Yup, also check how we do comments for structs elsewhere on the emu.

Reply:
There are 72 different colors, I don't really think it's a good idea to write all of these colors down in an enum.

Reply:
Oh lol... Fair enough then :)